### DISCUSSION TITLE
antipiccione

### DISCUSSION THREAD

**Discussion by lele-65** [indomus/forum#1611] [HUB: Home Assistant]
buongiorno a tutti
vorrei creare un antipiccione da installare in terrazza. Mi spiego meglio.
Vorrei utilizzare l'uscita audio del raspberry che ogni mattina ad un ora impostata si attivi ed ogni 15 minuti mi attivi un file audio.
Mi serve un indicazione  su dove poter iniziare a studiare come fare
Grazie

**Comment by Nik71git** [comment]
La soluzione migliore è una Air Arms S410 con una buona ottica, almeno una 3-9x32
